adapters/handlers/rest/handlers_objects.go

```diff
@@ -103,6 +103,16 @@ func (h *objectHandlers) validateObject(params objects.ObjectsValidateParams,
 }
 
 func (h *objectHandlers) getObject(params objects.ObjectsGetParams,
+	principal *models.Principal) middleware.Responder {
+	ps := objects.ObjectsClassGetParams{
+		HTTPRequest: params.HTTPRequest,
+		ID:          params.ID,
+		Include:     params.Include,
+	}
+	return h.findOne(ps, principal)
+}
+
+func (h *objectHandlers) findOne(params objects.ObjectsClassGetParams,
```

Comment:
since all other methods are called `_action_Object` I would keep `getObject` name for consistency and rename old method (now deprecated one) to something like `getObjectNoClass`. In case deprecated method will be removed in the future, api will remain consistent.

same applies for `deleteObject` and `deleteOne`

Reply:
Good point @aliszka I will consider it.

Reply:
Done